server/server.js

```diff
@@ -56,35 +60,38 @@ exports = {
     }
     issue_priority = priorityMap[`${priority_no}`];
 
-   find(args.data.ticket.id).then( data => {
-      const issueID = data.issue_data.issueID;
-      $request.put(`https://api.bitbucket.org/2.0/repositories/${args.iparams.bitBucket_repo}/issues/${issueID}`,{
-        headers: {
-          Authorization : "Bearer <%= access_token %>"
-        },
-        isOAuth: true,
-        json: {
-          "priority" : issue_priority
-        }
+    try {
+        const data = await find(args.data.ticket.id)
+        if(!data) throw new Error(`Error: couldn't able to find issue id, ${data}`);
+        const issueID = data.issue_data.issueID;
```

Comment:
Use [destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment)
```suggestion
        const { issueID } = data.issue_data;
```